src/VsixV3/ProjectTemplates1.x/Microsoft.VisualStudio.NetCore.ProjectTemplates.1.x.vsmanproj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I guess that's fine. We should make sure there's a packagegroup on the VS side that puts these together so that these go together.